sec_certs/certificate/common_criteria.py

```diff
@@ -185,14 +185,18 @@ def cert_id(self) -> Optional[str]:
             return processed if (processed := self.processed_cert_id) else self.keywords_cert_id
 
     @dataclass
-    class Heuristics(ComplexSerializableType):
+    class CCHeuristics(ComplexSerializableType):
         extracted_versions: List[str] = field(default=None)
         cpe_matches: Optional[List[Tuple[float, CPE]]] = field(default=None)
         labeled: bool = field(default=False)
         verified_cpe_matches: Optional[Set[CPE]] = field(default=None)
         related_cves: Optional[List[CVE]] = field(default=None)
         cert_lab: Optional[List[str]] = field(default=None)
         cert_id: Optional[str] = field(default=None)
+        directly_affected_by: Optional[List] = field(default=None)
+        indirectly_affected_by: Optional[Set] = field(default=None)
+        directly_affecting: Optional[Set] = field(default=None)
+        indirectly_affecting: Optional[Set] = field(default=None)
```

Comment:
It would be nice to have the type of the members of `List`, `Set` available. (Just guessing) `Optional[List[str]]` or `Optional[Set[str]]`?